content/blog/2020/10/09/FacilityLocationProblemQuickstart.adoc

```diff
@@ -6,7 +6,7 @@ yurloc
 :jbake-tags: use case, facility location problem, quarkus
 :jbake-social_media_share_image: facilityLocationProblemSolved.png
 
-We have created a new https://github.com/kiegroup/optaplanner-quickstarts/tree/development/quarkus-facility-location[quickstart] that extends OptaPlanner use case portfolio with the https://en.wikipedia.org/wiki/Facility_location_problem[facility location problem] and also demonstrates building a cloud-native application using https://quarkus.io/[Quarkus], the new Java Native stack.
+We have created a new https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/facility-location[quickstart] that extends OptaPlanner use case portfolio with the https://en.wikipedia.org/wiki/Facility_location_problem[facility location problem] and also demonstrates building a cloud-native application using https://quarkus.io/[Quarkus], the new Java Native stack.
```

Comment:
This was already merged https://github.com/kiegroup/optaplanner-website/pull/371. Maybe try rebasing your PR.